osmaxx-py/excerptconverter/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# scripts with names of form 'test*.py' are autodiscovered
```

Comment:
It's your code or the code of @das-g I copied from the other test module and I think it makes sense, because it's not the normal way python works. "tests" needs to be a python module but it's not necessary to import the tests = it's not the way python works normal.